src/main/java/racinggame/domain/RacingCar.java

```diff
@@ -0,0 +1,51 @@
+package racinggame.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import racinggame.view.RacingCarView;
+
+public class RacingCar {
+
+    private static final int MINIMUM_FORWARD_VALUE = 4;
+
+    private final RacingCarView racingCarView = new RacingCarView(this);
+    private String carName;
+    private int carPosition = 0;
+
+    private RacingCar() {}
+
+    public RacingCar(String carName) {
+        this.carName = carName;
+    }
+
+    public RacingCar(String carName, int position) {
+        this.carName = carName;
+        this.carPosition = position;
+    }
+
+    public static List<RacingCar> createRacingCarFromCarNames(String[] carNames) {
+        return Arrays.stream(carNames).map(RacingCar::new)
+                .collect(Collectors.toList());
+    }
```

Comment:
생성자를 체이닝으로 묶고, [정적 팩토리 메서드](https://velog.io/@ljinsk3/%EC%A0%95%EC%A0%81-%ED%8C%A9%ED%86%A0%EB%A6%AC-%EB%A9%94%EC%84%9C%EB%93%9C%EB%8A%94-%EC%99%9C-%EC%82%AC%EC%9A%A9%ED%95%A0%EA%B9%8C)로 객체를 생성하도록 수정해보는 것은 어떨까요? (`List<RacingCar>`생성을 명시적으로 표현할 수 있습니다.)

추가적으로 코드의 응집도를 위해 `Validator`에 있는 유효값 체크 로직을 다음과 같이 객체의 생성시점에 객체 체크해보는 것은 어떨까요?

```suggestion
    private RacingCar(String carName) {
        validateNameLength(carName);
        this(carName, INIT_POSITION);
    }

    private RacingCar(String carName, int position) {
        this.carName = carName;
        this.carPosition = position;
    }

    public static List<RacingCar> racingCarOf(String[] carNames) {
        return Arrays.stream(carNames).map(RacingCar::new)
                .collect(Collectors.toList());
    }
    
    private void validateNameLength(String carName) {
        if (carName.length() > MAXIMUM_NAME_LENGTH) {
            throw new IllegalArgumentException("자동차 이름의 글자 수는 5자 이하입니다.");
        }
    }
```

Reply:
생성자를 체이닝하니 필드에서 값을 초기화하지 않을 수 있어 좋은 것 같습니다.
응집성이 떨어지는 것 같아 검증 로직을 옮겨야하나 고민 했는데 감사합니다!

정적 팩토리 메서드에도 네이밍 컨벤션이 있었네요. 감사합니다!